### ISSUE TITLE
[idea] Fix accessibility violation "non-semantic-control" at div:nth-of-type(1) > div:nth-of-type(1) > nav:nth-of-type(1) > div:nth-of-type(1) > button:nth-of-type(1) > span:nth-of-type(1)

### ISSUE THREAD

**Issue by nhunsaker** [metatoy/sorb-demo#4]
### Intent

Fix accessibility violation "non-semantic-control" at div:nth-of-type(1) > div:nth-of-type(1) > nav:nth-of-type(1) > div:nth-of-type(1) > button:nth-of-type(1) > span:nth-of-type(1)

### Success criteria

Re-running the a11y audit against https://demo.sorbcloud.com/checkout no longer reports "non-semantic-control" at div:nth-of-type(1) > div:nth-of-type(1) > nav:nth-of-type(1) > div:nth-of-type(1) > button:nth-of-type(1) > span:nth-of-type(1)

### Altitude

ticket

### Source

brainstorm:a11y-20260816-015841
